control/veda_file_ingest.py

```diff
@@ -76,90 +77,67 @@ def __init__(self, course_object, video_proto, **kwargs):
         self.full_filename = kwargs.get('full_filename', None)
         self.complete = False
         self.archived = False
+        self.studio_upload_id = kwargs.get('studio_upload_id', None)
+
+    def ingest_from_s3(self):
+        """
+        Ingests a video from S3. Steps:
+        1 - Download video to node working directory from S3.
+        2 - Set up an ingest instance and insert video to ingestion phase.
+        3 - Move the video to 'processed' directory in s3.
+        """
+        filename = self.video_proto.s3_filename
+        file_downloaded = self._download_video_to_working_directory(self.studio_upload_id, filename)
+
+        if not file_downloaded:
```

Comment:
Shouldn't the scenario where `file_downloaded` is false be an error condition?

Reply:
I'm not sure. This method is just a rework of some of the content from the old validate_metadata_and_feed_to_ingest method. The equivalent line is [here](https://github.com/edx/edx-video-pipeline/pull/116/files/afa2f0e0632427713d185e8f96ec898a27565de8#diff-9158dcec44a33a59e532b52e766c6f12L304).